### PR TITLE
[HTML] Use connection icons and toolbar

### PR DESCRIPTION
**Pull request by m273d15**
* Introduces saros icons 
* Used icons to show connection state
* Used previously used message as tooltip

### CODE REVIEW

File: de.fu_berlin.inf.dpp.ui.frontend/html/src/components/ToolBar/ConnectionSplitButton.jsx [hunk 1]
```diff
@@ -14,37 +15,39 @@ class ConnectionSplitButton extends React.Component {
   constructor (props) {
     super(props)
     this.connectionStateMap = {
-      'INITIALIZING': { onClick: this.props.onConnect, messageId: 'action.connect' },
-      'NOT_CONNECTED': { onClick: this.props.onConnect, messageId: 'action.connect' },
-      'CONNECTED': { onClick: this.props.onDisconnect, messageId: 'action.disconnect' },
-      'ERROR': { onClick: noop, messageId: 'action.connectionError' },
-      'CONNECTING': { onClick: noop, messageId: 'action.connecting' },
-      'DISCONNECTING': { onClick: noop, messageId: 'action.disconnecting' }
+      'INITIALIZING': { onClick: this.props.onConnect, messageId: 'action.connect', icon: images.accountDisconnectedIcon },
```
Comment (m273d15) [inline de.fu_berlin.inf.dpp.ui.frontend/html/src/components/ToolBar/ConnectionSplitButton.jsx]: @AbdBarho I was not sure whether this creates a map containing the images as base64-string or simply as reference.

File: de.fu_berlin.inf.dpp.ui.frontend/html/webpack.config.js [hunk 2]
```diff
@@ -64,7 +64,17 @@ module.exports = (env = {}) => {
...
+        {
+          test: /\.(png|jp(e*)g)$/,
+          use: [{
+            loader: 'url-loader',
+            options: {
+              limit: 8000, // Convert images < 8kb to base64 strings
+              name: 'images/[hash]-[name].[ext]'
+            }
+          }]
+        }
```
[the remark this answers is not in the document]
Comment (AbdBarho) [reply de.fu_berlin.inf.dpp.ui.frontend/html/webpack.config.js]: maybe not all of it... are there any eot, woff, ttf or svg files that are being used in the HTML GUI?

File: de.fu_berlin.inf.dpp.ui.frontend/html/webpack.config.test.js [hunk 3]
```diff
@@ -28,6 +28,16 @@ module.exports = {
...
+      {
+        test: /\.(png|jp(e*)g)$/,
+        use: [{
+          loader: 'url-loader',
+          options: {
+            limit: 8000, // Convert images < 8kb to base64 strings
+            name: 'images/[hash]-[name].[ext]'
+          }
+        }]
+      }
```
Comment (AbdBarho) [inline de.fu_berlin.inf.dpp.ui.frontend/html/webpack.config.test.js]: keep in mind, [mocha-webpack](https://www.npmjs.com/package/mocha-webpack#which-version-works-with-mocha-webpack) only supports webpack up to version 3 and is no longer actively maintained, at some point we should switch to another solution.

Comment (m273d15) [reply de.fu_berlin.inf.dpp.ui.frontend/html/webpack.config.test.js]: Thanks for the hint. I think it is necessary to update the whole build infrastructure and several packages.

**Pull request merged**